pkg/ccip/bindings/feequoter/fee_quoter.go

```diff
@@ -300,3 +278,84 @@ type UpdateDestChainConfigs struct {
 	_       tlb.Magic                               `tlb:"#29950BAA"` //nolint:revive // Ignore opcode tag
 	Updates common.SnakeData[UpdateDestChainConfig] `tlb:"^"`
 }
+
+// binding types that supports FetchResult interface with rpc client
+
+type StaticConfig struct {
+	MaxFeeJuelsPerMsg  *big.Int
+	LinkToken          *address.Address
+	StalenessThreshold uint32
+}
+
+func (s *StaticConfig) FromResult(result *ton.ExecutionResult) error {
+	maxFeeJuelsPerMsg, err := result.Int(0)
+	if err != nil {
+		return err
+	}
+	linkTokenAddressSlice, err := result.Slice(1)
+	if err != nil {
+		return err
+	}
+	linkTokenAddress, err := linkTokenAddressSlice.LoadAddr()
+	if err != nil {
+		return err
+	}
+	tokenPriceStalenessThreshold, err := result.Int(2)
+	if err != nil {
+		return err
+	}
+	*s = StaticConfig{
+		MaxFeeJuelsPerMsg:  maxFeeJuelsPerMsg,
+		LinkToken:          linkTokenAddress,
+		StalenessThreshold: uint32(tokenPriceStalenessThreshold.Uint64()), //nolint:gosec // G115
+	}
+	return nil
+}
+
+func (s *StaticConfig) FetchResult(ctx context.Context, client ton.APIClientWrapped, block *ton.BlockIDExt, contractAddr *address.Address, opts *common.FetchOptions) error {
+	result, err := client.RunGetMethod(ctx, block, contractAddr, StaticConfigGetter)
+	if err != nil {
+		return fmt.Errorf("error getting staticConfig: %w", err)
+	}
+
+	if err = s.FromResult(result); err != nil {
+		return fmt.Errorf("failed to parse staticConfig: %w", err)
+	}
+
+	return nil
+}
+
+// FetchDestChainConfigsView fetches all destination chain configurations from the fee quoter contract
+func FetchDestChainConfigsView(ctx context.Context, client ton.APIClientWrapped, block *ton.BlockIDExt, feeQuoter *address.Address) (map[uint64]DestChainConfig, error) {
```

Comment:
This is a view generator, not directly a contract binding but an composition of multiple binding calls. I would separate this into a different layer (view), and keep bindings clean with only types/getters exposed by the contract.

Then once we code-gen the bindings we just replace that layer.